airflow/config_templates/config.yml

```diff
@@ -805,13 +805,15 @@ logging:
       version_added: 2.0.0
       type: string
       example: "{{ti.dag_id}}-{{ti.task_id}}-{{execution_date}}-{{try_number}}"
+      is_template: true
```

Comment:
Better way to say which fileds contain templates (used be endswith("_template").